drop-transfer/src/manager.rs

```diff
@@ -961,11 +961,31 @@ pub(crate) async fn resume(
     stop: &CancellationToken,
 ) {
     state.storage.cleanup_garbage_transfers().await;
```

Comment:
Let's move it to the beginning of the `restore_transfers_state()` so that we do not restore transfers to be deleted

Reply:
I've moved it outside into service::start() just before the restoration